feature-libs/cart/import-export/core/model/import-export.model.ts

```diff
@@ -0,0 +1,13 @@
+import { CmsComponent } from '@spartacus/core';
+
+export enum CartTypes {
+  NEW_SAVED_CART,
+  EXISTING_SAVED_CART,
+  ACTIVE_CART,
+}
+
+export interface CmsImportExportComponent extends CmsComponent {
+  importButtonDisplayPages: string[];
+  exportButtonDisplayPages: string[];
+  routesCartMapping: Map<string, CartTypes>;
```

Comment:
Can we use a plain object here (`Record<string, CartTypes>`) instead of Map? Arguments:
- `deepMerge()` for the config chunks doesn't support merging Maps
- there is no big advantage of using Map over a plain object in this case IMHO

Reply:
Oh, I didn't know it wouldn't work with deep merge. I wanted to use the native typing in this case, but I can do typing for an object as well